wcfsetup/install/files/lib/system/cache/eager/data/LanguageCacheData.class.php

```diff
@@ -62,6 +60,68 @@ public function getLanguageCategory(string $categoryName): ?LanguageCategory
      */
     public function getLanguageCategoryByID(int $languageCategoryID): ?LanguageCategory
```

Comment:
After checking the `LanguageFactory`, I think it’s safe to drop the `*Language*` from the methods for categories. It’s the language cache, what else should `getCategory()` return? :-)